client-app/src/Components/Programs.tsx

```diff
@@ -0,0 +1,57 @@
+import React, { useState, useEffect } from 'react';
+import { Link } from 'react-router-dom';
+import axios from 'axios';
+import '../css/Programs.css';
+import ProgramCard from './ProgramCard'; // Import the ProgramCard component
+
+interface Program {
+    id: number;
+    name: string;
+    description: string;
+    startDate: string;
+    aimAndCause: string;
+}
+
+function Programs() {
+    const [programs, setPrograms] = useState<Program[]>([]);
+    const [error, setError] = useState<string | null>(null);
+
+    useEffect(() => {
+        const fetchPrograms = async () => {
+            try {
+                const response = await axios.get<Program[]>(
+                    `${process.env.REACT_APP_BACKEND_API_BASE_URL}program`
+                );
+                setPrograms(response.data);
+            } catch (err) {
+                setError('Error fetching programs');
```

Comment:
It is VERY likely that it is because PGAdmin is not functioning on my end, but I get the "Error fetching programs" when I try to look at the programs page. With that said, I interestingly see both the error and "No programs available." It may be worth looking into how to only display the error when it comes up.